### DISCUSSION TITLE
Documentation for 'zeta' inline completion provider

### DISCUSSION THREAD

**Discussion by KhazAkar** [zed-industries/zed#22401] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I've seen that 'zeta' was added, and according to the initialization PR it's 'staff only'. it would be awesome to have documentation for this feature. Can we, users, use it somehow? If yes, how?

### Zed Version and System Specs

Zed: v0.166.1 (Zed)
OS: Linux Wayland ubuntu 24.04
Memory: 15.5 GiB
Architecture: x86_64
GPU: Intel(R) HD Graphics 5500 (BDW GT2) || Intel open-source Mesa driver || Mesa 24.0.9-0ubuntu0.3

**Comment by SomeoneToIgnore** [comment]
As well noted, it's not public yet so I doubt any documentation will appear before it gets public.

Even after that, I have doubts that it will be documented that much, as it's a similar Copilot/Supermaven/etc. idea of an inline assistant — you have to enable and follow the regular "proposed changes" flow as with other tools.

Seems pretty intuitive and with a well-visible knob to turn that kind of completion provider on, it might be fine without the docs.

**Comment by KhazAkar** [comment]
@SomeoneToIgnore if it's not yet public, why it's available to set in stable release of Zed then? I saw in initial PR some prediction keys etc and currently it works like some prediction in popup for autocomplete. If it will stay to work that way, that would be awesome :)

**Reply by SomeoneToIgnore** [reply]
I cannot explain that, unfortunately, can only guess that it's a mistake, cc @mrnugget 
I am only aware that there are no plans to make it public right now, as there are bugs and workflow improvements to iron out — due to that I can not guarantee that anything stays the way it is in Stable now.

**Reply by mrnugget** [reply]
> if it's not yet public, why it's available to set in stable release of Zed then? I

What do you mean with "avialable"? We don't advertise it anywhere, right? You can change your settings manually to use it, but it won't work if you're not a Zed staff member, right?

**Reply by KhazAkar** [reply]
@mrnugget available as in selectable :)
And yes, it's not advertised anywhere, just found it and without any special clearance I have slightly different autocomplete experience than with "none" selected

**Reply by mrnugget** [reply]
> available as in selectable :)

In the settings.json you mean, right? Or somewhere else?

**Reply by KhazAkar** [reply]
Yes, in settings.json
I was just curious :)

**Reply by mrnugget** [reply]
Okay, yeah. I don't think that's a problem. We'll want to release it soon. Until then I think this papercut is acceptable.

**Reply by KhazAkar** [reply]
Even in current form, idk if it's even that, seems pretty cool to be honest.
Rooting hard for Zed :)
